### PR TITLE
Propagating CGE-predicted phenotypes from Resfinder to the summary outputs

### PR DESCRIPTION
**Pull request by emarinier**
Propagates the Resfinder CGE-predicted phenotypes into the summary and detailed summary.

### CODE REVIEW

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. Thanks so much Eric 😄 .

One thing I did notice, which I think might be unrelated to this particular PR but is related to adding the **CGE Predicted Phenotype**  is that some of the values show up as "unknown" even though they appear to exist in the ResFinder database.

For example, in the attached fasta file, [GCF_000008105.1_ASM810v1_genomic.fna.zip](https://github.com/phac-nml/staramr/files/11266503/GCF_000008105.1_ASM810v1_genomic.fna.zip), I get a **unknown[cmlA1_1_M64556]** result when running staramr.

| Isolate ID | Gene | Predicted Phenotype | CGE Predicted Phenotype | %Identity | %Overlap | HSP Length/Total Length | Contig | Start | End | Accession |
|-|-|-|-|-|-|-|-|-|-|-|
GCF_000008105.1_ASM810v1_genomic | cmlA1 | chloramphenicol | **unknown[cmlA1_1_M64556]** | 99.92 | 100.00 | 1260/1260 | ref\|NC_006856.1\| | 41866 | 40607 | M64556 |

However, looking up the following accession, **M64556**, in the ResFinder database:

https://bitbucket.org/genomicepidemiology/resfinder_db/src/fa32d9a3cf0c12ec70ca4e90c45c0d590ee810bd/phenotypes.txt#lines-2297

I get a Phenotype of **Amphenicol**.

I would expect to see this same phenotype, **Amphenicol**, in the StarAMR results table instead of **unknown[cmlA1_1_M64556]**.

Would you have an idea why this is happening?

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. I did a comparison of results with ~1300 campylobacter genomes and the information all seems consistent (only the **CGE Predicted Phenotype** column is added).

Just one comment.

File: staramr/tests/unit/results/test_AMRDetectionSummaryResistance.py [hunk 1]
```diff
@@ -19,15 +19,16 @@ def setUp(self):
...
              '741/741', 'blaIMP-42_1_AB753456', 1, 741, 'AB753456']
         ],
             columns=self.columns_resfinder)
 
         self.resfinder_table_duplicate_resistances = pd.DataFrame([
-            ['file1', 'blaIMP-42', 'ampicillin', 99.73, 100.00,
+            ['file1', 'blaIMP-42', 'ampicillin', 'ampicillin', 99.73, 100.00,
              '741/741', 'blaIMP-42_1_AB753456', 1, 741, 'AB753456'],
-            ['file1', 'blaCTX-M-55', 'ampicillin, ceftriaxone', 99.73, 100.00,
+            ['file1', 'blaCTX-M-55', 'ampicillin, ceftriaxone', 'ampicillin, ceftriaxone', 99.73, 100.00,
```
[the remark this answers is not in the document]
Comment (emarinier) [reply staramr/tests/unit/results/test_AMRDetectionSummaryResistance.py:31]: I believe this is now addressed in [23b9bc9](https://github.com/phac-nml/staramr/pull/173/commits/23b9bc9a81733a11985cc11c8e2dd7ead38e10b5).

**Comment by emarinier** [comment]
As per request, the changelog has been updated.

**Review by apetkau — APPROVED** [review]
Thanks so much. This all looks great to me. Merging.

**Pull request merged**